TESTS/host_tests/pyusb_basic.py

```diff
@@ -45,22 +51,189 @@ def get_interface(dev, interface, alternate=0):
 VENDOR_TEST_CTRL_NONE_DELAY = 6
 VENDOR_TEST_CTRL_IN_STATUS_DELAY = 7
 VENDOR_TEST_CTRL_OUT_STATUS_DELAY = 8
+VENDOR_TEST_CTRL_IN_SIZES = 9
+VENDOR_TEST_CTRL_OUT_SIZES = 10
 VENDOR_TEST_UNSUPPORTED_REQUEST = 32
 
+REQUEST_GET_STATUS = 0                  # done
+REQUEST_CLEAR_FEATURE = 1               # done
+REQUEST_SET_FEATURE = 3                 # done
+REQUEST_SET_ADDRESS = 5                 # ???
```

Comment:
Does the ??? correspond to some sort of status?

Reply:
It's for my private use :)
Will be removed soon